simulation_parameters/industrial_stage/units.json

```diff
@@ -3,7 +3,18 @@
     "Name": "UNIT_SIMPLE_ROCKET",
     "UnitIcon": "res://assets/textures/gui/bevel/technologies/UnknownTechnology.png",
     "VisualScene": "res://assets/models/space/PlaceholderRocket.tscn",
-    "SpaceVisuals": "res://assets/models/space/PlaceholderRocketSpace.tscn",
+    "SpaceVisuals": "res://assets/models/space/PlaceholderAdvancedSpaceship.tscn",
```

Comment:
Is this space visuals override here specifically needed still? I though the code change would make this work without having to hack the unit to look way different, which is a hidden bug now that will trigger at some point in the future.

Reply:
I didn't manage to get it working any other way, so that line is solely determining what appears in the space stage.

Reply:
This should be reverted anyway, I can help with the code implementation once the graphics files have been cleaned up.